src/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.markdown

```diff
@@ -283,7 +285,7 @@ Email content to: [concur_AppCenterMarketing@sap.com](mailto:concur_AppCenterMar
 
 * App Center Alliances: [concur_AppCenterAlliance@sap.com](mailto:concur_AppCenterAlliance@sap.com)
 * App Center Marketing: [concur_AppCenterMarketing@sap.com](mailto:concur_AppCenterMarketing@sap.com)
-* App Center Channel Sales: [aaron.guerra@sap.com (SMN)](mailto:aaron.guerra@sap.com) and [arian.white@sap.com (ENT)](mailto:arian.white@sap.com)
+* App Center Channel Sales: [aaron.guerra@sap.com (SMN)](mailto:aaron.guerra@sap.com) and TBH - ENT
```

Comment:
We need to remove individual email names and replace them with distribution lists and no TBH -- just leave out.